pkg/controllers/networkinfo/networkinfo_controller.go

```diff
@@ -146,6 +139,15 @@ func (r *NetworkInfoReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			log.Error(err, "failed to get NSX VPC ConnectivityProfile object", "vpcConnectivityProfileName", vpcConnectivityProfileName)
 			return common.ResultRequeue, err
 		}
+
+		if ncName == commonservice.SystemVPCNetworkConfigurationName {
+			if len(vpcConnectivityProfile.ExternalIpBlocks) == 0 {
+				setVPCNetworkConfigurationStatusWithNoExternalIPBlock(ctx, r.Client, vpcNetworkConfiguration, false)
+				log.Error(err, "there is no ExternalIPBlock in VPC ConnectivityProfile", "VPC", req.NamespacedName)
```

Comment:
Just want to double confirm that we don't need to return for the error, right?

Reply:
https://github.com/vmware-tanzu/nsx-operator/pull/730#discussion_r1734235894